### ISSUE TITLE
Video uploads but doesnt play

### ISSUE THREAD

**Issue by qota03** [546200350/TikTokUploder#10]
after uploading, the video doesnt play but the audio does

**Comment by 546200350** [comment]
is something wrong with your origin video?
maybe you can try upload by website,make sure the video is ok

**Comment by qota03** [comment]
yeah i uploaded manually with the same file and it works but when I use this to upload the video freeze but the audio still plays

**Comment by 546200350** [comment]
can you send the video to me ?
